rqd/rqd/rqcore.py

```diff
@@ -802,9 +802,10 @@ def launchFrame(self, runFrame):
     def getRunningFrame(self, frameId):
         try:
             return self.__cache[frameId]
-        except:
-            raise KeyError("frameId {} is not running on this"
-                           "machine".format(frameId))
+        except Exception:
+            log.info("frameId {} is not running on this"
+                     "machine".format(frameId))
```

Comment:
I think this can fit on a single line. (We haven't written down an exact coding style yet but a hard 80-char limit is probably overkill IMO.)